### ISSUE TITLE
Problem updating to 10.0.2 RC1 with the Market app

### ISSUE THREAD

**Issue by settermjd** [owncloud/core#28052]
### Steps to reproduce
1. Run the upgrade from the web UI to update from 10.0.1 to 10.0.2 RC1

### Expected behaviour

I would have expected to have had a smooth upgrade with everything work. 

### Actual behaviour

The UI shows the following error:

```
Repair warning: Market app is unavailable for updating of apps. Enable with: occ:app:enable market
```

Expanding the detailed logs, the following information is listed:

```
Preparing update
Set log level to debug
Turned on maintenance mode
Repair warning: Market app is unavailable for updating of apps. Enable with: occ:app:enable market
Updating database schema
Updated database
Updated "dav" to 0.2.9
[1 / 28]: Drop old database table: calendar_calendars
[2 / 28]: Drop old database table: calendar_objects
[3 / 28]: Drop old database table: calendar_share_calendar
[4 / 28]: Drop old database table: calendar_share_event
[5 / 28]: Drop old database table: file_map
[6 / 28]: Drop old database table: foldersize
[7 / 28]: Drop old database table: fscache
[8 / 28]: Drop old database table: gallery_sharing
[9 / 28]: Drop old database table: locks
[10 / 28]: Drop old database table: log
[11 / 28]: Drop old database table: media_albums
[12 / 28]: Drop old database table: media_artists
[13 / 28]: Drop old database table: media_sessions
[14 / 28]: Drop old database table: media_songs
[15 / 28]: Drop old database table: media_users
[16 / 28]: Drop old database table: permissions
[17 / 28]: Drop old database table: pictures_images_cache
[18 / 28]: Drop old database table: principalgroups
[19 / 28]: Drop old database table: principals
[20 / 28]: Drop old database table: queuedtasks
[21 / 28]: Drop old database table: sharing
[22 / 28]: Drop old database table: clndr_calendars
[23 / 28]: Drop old database table: clndr_objects
[24 / 28]: Drop old database table: clndr_share_event
[25 / 28]: Drop old database table: clndr_share_calendar
[26 / 28]: Drop old database table: clndr_repeat
[27 / 28]: Drop old database table: contacts_addressbooks
[28 / 28]: Drop old database table: contacts_cards
[28 / 28]: Drop old database tables
[1 / 2]: Fix permissions so avatars can be stored again
[1 / 1]: Fix permissions so avatars can be stored again
Turned off maintenance mode
Reset log level
```

The specific problem that I have is that if I attempt to run the suggested command, which is `occ:app:enable market` the command is not found. If I change it to `./occ app:enable market` the output is: `market not found`.

### Server configuration

**Operating system:** Ubuntu 16.04 LTS

**Web server:** Apache 2

**Database:** MySQL

**PHP version:** 7.1.5

**ownCloud version:** 10.0.2 RC1 after upgrade / 10.0.1 before upgrade

**Updated from an older ownCloud or fresh install:** Upgrade

**Where did you install ownCloud from:** Downloaded the source

**Are you using external storage, if yes which one:** 

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** 

### Client configuration

**Browser:** Safari

**Operating system:** macOS Sierra

**Comment by PVince81** [comment]
from the web UI ? that would be the web updater.

Can you check if the folder "apps/market" exists ? If it doesn't then something is very wrong...

cc @VicDeo

**Comment by phil-davis** [comment]
Note: The problem with the extra colon in the suggested occ command claims to be fixed in 10.0.2 by https://github.com/owncloud/core/commit/3c88e931dc47094d4d6baf1d6b42c1eb41833462
So I guess that little text fix was not in 10.0.2RC1

**Comment by PVince81** [comment]
the little text fix was "sneaked" into 10.0.2...

**Comment by PVince81** [comment]
closing due to lack of feedback
